READMEs/install.md

```diff
@@ -55,17 +55,9 @@ Issue: MacOS "Unsupported Architecture"
 Issue: Dependencies and Python 3.11
 
 - ocean.py has two dependencies that don't support Python 3.11 yet. Here's the workaround for each.
-- (1) The `eth-brownie` package uses Vyper which doesn't support Python 3.11. The workaround: before installing ocean-lib, run `pip install vyper==0.3.7 --ignore-requires-python` and `sudo apt-get install python3.11-dev`
+- (1) Vyper which doesn't support Python 3.11. The workaround: before installing ocean-lib, run `pip install vyper==0.3.7 --ignore-requires-python` and `sudo apt-get install python3.11-dev`
```

Comment:
Why was this line kept at all? Afaik, it was only Brownie that uses Vyper. If that's correct, we can remove the line.

Reply:
You are right, I removed it.